src/App.tsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+
+import { FullTodo, Todo } from './components/types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+function preparedTodos(
```

Comment:
Можна цю функцію винести в окремий файл `helpers.ts`
```suggestion
function prepareTodos(
```